internal/engine/upper.go

```diff
@@ -92,6 +92,8 @@ func (u Upper) CreateManifests(ctx context.Context, manifests []model.Manifest,
 		if err == nil {
 			buildStates[manifest.Name] = NewBuildState(buildResult)
 			lbs = append(lbs, k8s.ToLoadBalancerSpecs(buildResult.Entities)...)
+		} else if isPermanentError(err) {
```

Comment:
does this error get surfaced to the user in a useful way?

Reply:
yes